### ISSUE TITLE
UBports installer fails to install \w no helpful message: Adb push error: 

### ISSUE THREAD

**Issue by stefanreimers** [ubports/ubports-installer#213]
Automatically generated error report 
Version: 0.1.12-beta 
Host OS: win32 
Host Dist: undefined undefined
Host Arch: ia32 
Node: v7.4.0 

Error log: http://past

===
Some more context: I tried to install stable or dev build on a OnePlus One

**Comment by NeoTheThird** [comment]
The pasted URL seems to be broken, any chance you could get me the log file? Somewhere in `%APPDATA%\ubports`. Don't know the exact path on windows rn *blush*

**Comment by stefanreimers** [comment]
Hi,

sorry for the delay. I uploaded the log at https://gist.github.com/stefanreimers/48f94024d978d4ae1ebab9247a00634a. So far I tried:
1. using the Universal ADB Driver or the adb version found in the Android SDK
2. executing as normal user or Administrator
2. deleting cached images from disk

Nothing worked so far. Maybe its something with the device. It has an custom ROM already. When installation on Windows won't work I will use my Ubuntu device to setup my OnePlus One.
